### DISCUSSION TITLE
[x][closed] Small points from Kieron (copied from email)

### DISCUSSION THREAD

**Discussion by sveinugu** [fairtracks/fairtracks_standard#99] [Team Posts]
* I note there are \t at the start of the some EDAM IRIs in the file, which is undoubtedly bad.
e.g. "term_iri": "\thttp://edamontology.org/format_3006"

  * Is a JSON null better than "NA" as a string? 
```"tracks": [
    {
      "local_id": "NA",
```

**Comment by sveinugu** [comment]
@nerdstrike 
1. Yes a bug. Will fix it
2. Well there should be neither 'NA' or JSON Null, but a real local id. Without the local id, we are unable to link the different types of records together in TrackFind search. You could just make up one, but they must not change in newer versions of the metadata.

**Comment by dzerbino** [comment]
In this case, those are tracks that have local_id = "NA". While this is annoying, it is not really necessary since in this case there are no aggregated tracks so these IDs are never used (hence they were not generated)

**Comment by sveinugu** [comment]
I see. However there might be some issues with the versioning in TrackFind, e.g. how to go back in time for a record if you don’t have the local_id. @dtitov How is this handled now after the refactoring? We discussed the issue so much that I have forgotten the conclusions.. 😀

**Comment by dtitov** [comment]
@sveinugu, we will need `local_id` in tracks only for backward mapping (if we will produce our own tacks based on raw data tracks).

**Comment by dzerbino** [comment]
OK, I'll fudge local_ids, it's a fairly minor point.
